### PR TITLE
Internationalization date

### PR DESCRIPTION
**Pull request by edumoreira1506**
Fix internationalization date. Preview:
Peek 2019-11-25 13-30

### CODE REVIEW

File: app/assets/javascripts/components/sprints/Sprint.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    points: 0,
+    stories: []
+  }
+};
+
+const Sprint = ({
+  fetchStories,
+  sprint
+}) => {
+  const isDone = () => sprint.hasOwnProperty('hasStories');
```
Comment (nolleto) [inline app/assets/javascripts/components/sprints/Sprint.js]: You can use [`useMemo`](https://reactjs.org/docs/hooks-reference.html#usememo) here:
```js
const isDone = useMemo(() => sprint.hasOwnProperty('hasStories'), [sprint]);

isDone // true|false
```

Comment (edumoreira1506) [reply app/assets/javascripts/components/sprints/Sprint.js]: Gotcha! Thanks.

File: app/assets/javascripts/components/stories/Sprints.js [hunk 2]
```diff
@@ -10,18 +10,17 @@ const defaultProps = {
   sprints: [],
 };
 
-const renderSprints = (sprints, fetchStories) => {
-  return sprints.map(
-    (sprint, index) =>
+const renderSprints = (sprints, fetchStories) =>
+  sprints.map(
+    sprint =>
       sprint ? (
```
Comment (nolleto) [inline app/assets/javascripts/components/stories/Sprints.js]: There is a chance of a sprint to be null?
If so, I would filter first and then map them.
```js
const renderSprints = (sprints, fetchStories) =>
  sprints
    .filter(Boolean)
    .map(sprint => (
      ...
    ))
```

Comment (edumoreira1506) [reply app/assets/javascripts/components/stories/Sprints.js]: I believe that we can use just:
```
sprints.map(
    sprint =>
      <Sprint
        key={sprint.number}
        sprint={sprint}
        fetchStories={fetchStories}
      />
  )
```
I did some tests and search in the code and has no cases when sprint is null. thanks!

**Comment by talyssonoc** [comment]
Can we hide the time and show only the date? We don't need to show `00:00:00`.

**Comment by edumoreira1506** [comment]
Updated:
Peek 2019-12-03 09-50

**Pull request merged**